src/App.js

```diff
@@ -52,17 +52,28 @@ export default class App extends Lightning.Component {
     })
     await testIncreaseSplashVisibility
 
+    let date = new Date();
+    let time = date.getHours() + ":" + date.getMinutes();
+
```

Comment:
Clock is never updating, always shows the same time. Restart of application is required to update the clock.

Reply:
Addressed the review comments, please check